json.cc

```diff
@@ -42,13 +46,17 @@ static void json_write_str_value(FILE *out, const char *str)
     fputc('"', out);
 }
 
+static void json_write_name_value(FILE *out, const char *name)
+{
+    fprintf(out, "\"%s\":", name);
```

Comment:
I'd presume that you'd want to avoid printf and it's family. They are slow because they need to parse the input (i.e. `"\"%s\""`) and handle the arguments accordingly, at the same time making sure that there is sufficient amount of buffer.

It'd be much more lightweight to do `fputc('"'); fwrite(name, ...); fputc('"');`.

FWIW, https://github.com/h2o/qrintf might be a good resource to see what the ballpark figures are. (note: I'm not suggesting that we should be using qrintf).

Reply:
Looks good. Changed to use `fwrite` as much as possible. 14ec4b653020b6fd1b99e9c2946d655285367dbe